patches/com/mojang/blaze3d/platform/GlStateManager.java.patch

```diff
@@ -17,7 +17,7 @@
     }
  
     public static void _texParameter(int p_84332_, int p_84333_, int p_84334_) {
-@@ -945,5 +952,54 @@
+@@ -945,5 +_,54 @@
```

Comment:
These don't belong, I think?

Reply:
Those patches have been messed up by the lack of auto headers, they should be fixed at some point, it doesn't really matter when.